whisper/faster-whisper-v2/model/model.py

```diff
@@ -16,13 +17,39 @@ def load(self):
         self._model = WhisperModel(self._config["model_metadata"]["model_id"])
 
     def preprocess(self, request: Dict) -> Dict:
-        resp = requests.get(request["url"])
-        return {"response": resp.content}
+        audio_base64 = request.get("audio")
+        url = request.get("url")
+
+        if audio_base64 and url:
+            return {
+                "status": 400,
+                "msg": "Only a base64 audio file OR a URL can be passed to the API, not both of them!",
```

Comment:
nit: I think we should avoid exclamation points in these. Maybe something a little more neutral like:
"message": "Please provide either a base64-encoded audio file or a URL, but not both."

there may be enough here to warrant wrapping this in a little helper method that can be included in other models that take audio files as input